website/docs/api/matcher.md

```diff
@@ -44,6 +44,8 @@ rule-based matching are:
 | `SPACY`                                         | Token has a trailing space. ~~bool~~                                                                                      |
 |  `POS`, `TAG`, `MORPH`, `DEP`, `LEMMA`, `SHAPE` | The token's simple and extended part-of-speech tag, morphological analysis, dependency label, lemma, shape. ~~str~~       |
 | `ENT_TYPE`                                      | The token's entity label. ~~str~~                                                                                         |
+| `ENT_ID`                                      | The token's entity id (ent_id_). ~~str~~                                                                                         |
+| `ENT_KB_ID`                                      | The token's entity kb id (ent_kb_id_). ~~str~~                                                                                         |
```

Comment:
```suggestion
| `ENT_ID`                                        | The token's entity ID (`ent_id`). ~~str~~                                                                                 |
| `ENT_KB_ID`                                     | The token's entity knowledge base ID (`ent_kb_id`). ~~str~~                                                               |
```

(It's hard to get the formatting right in the suggestions. We can run `prettier` again later.)